docs/source/api.rst

```diff
@@ -1,5 +1,84 @@
 AA Toolbox API
 ==============
 
-.. automodule:: aatoolbox
+Data sources
+------------
+
+CHIRPS
+^^^^^^
+
+.. automodule:: aatoolbox.datasources.chirps.chirps
+   :members:
+   :undoc-members:
```

Comment:
Maybe it's not necessary to include `undoc-members`? Not sure we want to generate docs for functions with docstrings. (On a similar note, not sure we need `show-inheritance`, tbh I'm not sure what it does exactly)